src/applications/vaos/new-appointment/components/ConfirmationPage/ConfirmationDirectScheduleInfoV2.jsx

```diff
@@ -34,7 +34,7 @@ export default function ConfirmationDirectScheduleInfoV2({
         {` ${getTimezoneAbbrByFacilityId(data.vaFacility)}`}
```

Comment:
I don't think this file should be changed. This ticket is only for the vaccine confirmation page.

Reply:
Good catch, thank you. These should be reverted now